### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by bipindubey-technoark** [srigar/multiselect-react-dropdown#69]
I am using this component in nextjs version 9.1.1 and i added this using yarn add command but it is giving the error 
ReferenceError: window is not defined

**Comment by atulsuresh** [comment]
https://github.com/srigar/multiselect-react-dropdown/issues/47#issuecomment-661565310 should help

In short, NextJS tries to render the page on the server where _window_ doesn't exist. The easiest way to solve this importing the component as dynamic which will try to import the component while on the browser.

```javascript
import dynamic from 'next/dynamic';
const Multiselect = dynamic(
    () => import('multiselect-react-dropdown').then(module => module.Multiselect),
    {
        ssr: false
    }
)
```

**Comment by pragaut** [comment]
Thanks a lot,

you save my day ...

**Comment by srigar** [comment]
Duplicate #47.

**Comment by srigar** [comment]
It has been fixed in version >=2.0.0. Feel free to reopen issue if still persist
